apps/storybook/src/CalendarList.stories.tsx

```diff
@@ -14,7 +14,7 @@ export const Normal = () => (
   <CalendarList
     page={select<ComponentProps<typeof CalendarList>['page']>(
       'Page',
-      ['event', 'calendar', 'group'],
+      ['event', 'calendar', 'group', 'calendar-working-group'],
```

Comment:
Why is this calendar-working-group? Shouldn't it just be working-group?

Reply:
That will happen once we add the calendar list on the Working Group page, basically "calendar" is meant for the Calendar page, where we showed the Groups Calendar list. I think I should rather rename "calendar" to "calendar-group"

Reply:
Starting to wonder if you should just inject the title and description to simplify the whole thing. How many times is the component used? 4 times?